test/functional/test_framework/test_framework.py

```diff
@@ -130,8 +130,8 @@ def main(self):
         self.options.bitcoind = os.getenv("BITCOIND", default=config["environment"]["BUILDDIR"] + '/src/bitcoind' + config["environment"]["EXEEXT"])
         self.options.bitcoincli = os.getenv("BITCOINCLI", default=config["environment"]["BUILDDIR"] + '/src/bitcoin-cli' + config["environment"]["EXEEXT"])
 
-        os.environ['PATH'] = config['environment']['BUILDDIR'] + os.pathsep + \
-                             config['environment']['BUILDDIR'] + os.path.sep + "qt" + os.pathsep + \
+        os.environ['PATH'] = os.path.join(config['environment']['BUILDDIR'], 'src') + os.pathsep + \
```

Comment:
nit to avoid manual string building and os.pathsep duplicate

```
os.pathsep.join([
  os.path.join(config['environment']['BUILDDIR'], 'src'),
  os.path.join(config['environment']['BUILDDIR'], 'src', 'qt'),
  os.environ['PATH']
])
```